omc3/global_correction.py

```diff
@@ -47,7 +47,7 @@
 
 *--Optional--*
 
-- **beta_file_name**:
+- **beta_filename**:
```

Comment:
Is this gonna break something from the side of the GUI calling?

Reply:
yes, but I changed the `max_iter` to `iterations` as well, so I have updated that already in the branch I am working on